### PR TITLE
feat(stripe): upgrade api and sdk version

### PR DESCRIPTION
**Pull request by gary02**

### CODE REVIEW

File: src/routes/pay/stripe/index.ts [hunk 1]
```diff
@@ -18,10 +18,6 @@ import { createRefundTxs, updateTxState } from './transaction'
 
 const logger = getLogger('route-stripe')
 
-const stripe = new Stripe(environment.stripeSecret, {
-  apiVersion: '2020-08-27',
```
Comment (robertu7) [inline src/routes/pay/stripe/index.ts:22]: API version will be update-to-date with `stripe` package if we don't specify a `apiVersion`?

File: src/connectors/__test__/stripe.test.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+const testUser2 = {
+  id: '2',
+  email: 'test2@matters.news',
+}
+
+// const stripeAPI = (stripe as any).stripeAPI as Stripe
+let customer: Stripe.Customer
+
+test('create customer', async () => {
+  const _customer = await stripe.createCustomer({ user: testUser1 })
```
Comment (robertu7) [inline src/connectors/__test__/stripe.test.ts:20]: Is it run on [Stripe mock](https://stripe.com/docs/automated-testing)?

**Review by robertu7 — APPROVED** [review]

File: src/routes/pay/stripe/index.ts (hunk 1 above)
Comment (gary02) [reply src/routes/pay/stripe/index.ts:22]: start from v12,it  seem have to specify a apiVersion

File: src/connectors/__test__/stripe.test.ts (hunk 2 above)
Comment (gary02) [reply src/connectors/__test__/stripe.test.ts:20]: run on  https://github.com/stripe/stripe-mock

Comment (gary02) [reply src/connectors/__test__/stripe.test.ts:20]: https://github.com/thematters/matters-server/pull/3449/files#diff-dcd9bbd1ca93eef82a91d6b61ddfbf6502e4573161f62786d6ea188edeb1080bR41

**Pull request merged**